src/please-wait.coffee

```diff
@@ -245,6 +247,7 @@
         if animationSupport then @_loadingElem.removeEventListener(animationEvent, listener)
         # Reset the loading screen element since it's no longer attached to the DOM
         @_loadingElem = null
+        @_onFinishedCallback.apply(this)
```

Comment:
No problem - you are of course more than welcome to modify the source directly to suit your needs! I'm going to close the pull request for now then. If you want to make some changes to make this a bit more generically reusable, feel free to reopen at any time :+1: